src/effects/globals.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/prefer-default-export */
+import { db } from 'util/firebase';
+
+export function fetchCurrentTerm() {
+  return db.collection('terms').get()
+    .then(querySnapshot => querySnapshot.docs.reduce(
```

Comment:
Can you `orderBy` instead? https://firebase.google.com/docs/firestore/query-data/order-limit-data

Reply:
Done